packages/graphql-contentful-helpers/src/createContext.ts

```diff
@@ -37,34 +51,64 @@ export type CreateContextProps = {
 const createContext = async ({ config }: CreateContextProps): Promise<ApolloContext> => {
   const pathReaders = createPathReaders(config);
 
-  const locales = await getLocales(
-    config.contentful.spaceId,
-    config.contentful.env,
-    config.contentful.contentDeliveryToken
-  );
-
-  const defaultLocale = get(
-    find(locales, (locale) => locale.default),
-    'code',
-    'en-US'
-  );
-
-  const contentful = {
-    prod: createClient({
-      accessToken: config.contentful.contentDeliveryToken,
-      space: config.contentful.spaceId,
-      environment: config.contentful.env,
-      host: 'cdn.contentful.com',
-      resolveLinks: false
-    }),
-    preview: createClient({
-      accessToken: config.contentful.contentPreviewToken,
-      space: config.contentful.spaceId,
-      environment: config.contentful.env,
-      host: 'preview.contentful.com',
-      resolveLinks: false
-    })
-  };
+  let locales: string[] = [];
+  let defaultLocale: string = 'en-US';
+  let clients: { prod: any; preview: any };
+  let loaders;
+
+  if (config.cms === 'Sanity') {
+    const sanityCfg = (config as any).sanity || {};
+    const prodClient = sanityClient({
+      projectId: sanityCfg.projectId,
+      dataset: sanityCfg.dataset,
+      apiVersion: sanityCfg.apiVersion || '2021-03-25',
+      token: sanityCfg.token,
+      useCdn: !sanityCfg.usePreview
```

Comment:
useCdn is not really the same as usePreview in the contentful config. Also, useCdn was never defined in the AppConfig. We have to take a closer look and figure out how to do the preview/draft thing in sanity

Reply:
Actually, it seems that useCdn is used in combination with:
`perspective: 'drafts', `, so maybe this approach works.